solutions/client-landing-zone/client-folder/firewall-policy/rules/iap.yaml

```diff
@@ -14,6 +14,7 @@
 #########
 # Delegate to host project, ingress IAP traffic (firewall) from IP range 35.235.240.0/20 to VPC resources in shared VPC network
 # AC-3(9), AC-4, AC-4(21), SC-7(5), SC-7(8), SC-7(9), SC-7(11) - All connections to or from virtual machine instances are allowed/denied via firewall rules configured in shared VPC network within host project or firewall policies in parent folders based on least-privilege principle. Each firewall rule applies to incoming(ingress) or outgoing(egress) connections, not both.
+# AU-12 - Disable Logging for firewall policies
```

Comment:
I wondered why we disabled logging here until I read `# logging not supported for goto_next rules` on line 30. Any chance we can duplicate or move this comment after line 17? If so, there's also a bunch of other files that this comment would apply to.

Reply:
great suggestion thanks alain and luc. Removed au-12 tagging for go to next rules